lms/migrations/versions/f68aacfc62c7_migration_for_grading_sync_tables.py

```diff
@@ -0,0 +1,51 @@
+"""Migration for grading_sync tables
+
+Revision ID: f68aacfc62c7
+Revises: adc83819c8d8
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = 'f68aacfc62c7'
+down_revision = 'adc83819c8d8'
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('grading_sync',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('assignment_id', sa.Integer(), nullable=False),
+    sa.Column('status', sa.Enum('scheduled', 'in_progress', 'finished', 'failed', name='autogradingsyncstatus', native_enum=False, length=64), nullable=False),
+    sa.Column('created', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+    sa.Column('updated', sa.DateTime(), server_default=sa.text('now()'), nullable=False),
+    sa.ForeignKeyConstraint(['assignment_id'], ['assignment.id'], name=op.f('fk__grading_sync__assignment_id__assignment')),
+    sa.PrimaryKeyConstraint('id', name=op.f('pk__grading_sync'))
+    )
+    op.create_index(op.f('ix__grading_sync_assignment_id'), 'grading_sync', ['assignment_id'], unique=False)
+    op.create_table('grading_sync_grade',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('grading_sync_id', sa.Integer(), nullable=False),
+    sa.Column('lms_user_id', sa.Integer(), nullable=False),
+    sa.Column('grade', sa.Float(), nullable=True),
+    sa.Column('extra', postgresql.JSONB(astext_type=sa.Text()), server_default=sa.text("'{}'::jsonb"), nullable=False),
+    sa.Column('success', sa.Boolean(), nullable=True),
```

Comment:
Yeah we could use status instead of this, in this PR the status is:

Null grade & Null success, scheduled
Null grade & success = False, failed
grade & success = True, succeeded

Reply:
I don't have a strong opinion here. If you think using a boolean we can infer all required information, fine by me.